app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,13 @@
+from abc import abstractmethod
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    @abstractmethod

```

Comment:
The `@abstractmethod` decorator is not appropriate for the `__init__` method. Constructors should not be abstract because they are meant to initialize instances of the class. Consider removing the `@abstractmethod` decorator from the `__init__` method.